### DISCUSSION TITLE
Support PDF export for QR codes

### DISCUSSION THREAD

**Discussion by 0nullpointer** [ente-io/ente#500] [Enhancements]
Currently the user is able to view the QR of individual codes in the app. Supporting an export of PDF that contains account details, including QR code which users can print out is useful, which is a feature in Raivo.

**Comment by ghost** [comment]
Regarding exports, Raivo also supports ZIP-exports. So, perhaps adding support for this as well would be worth considering.

**Comment by ghost** [comment]
Any reason the original [Feature Request] Support PDF export for QR codes #460 was closed?

**Reply by mnvr** [reply]
Hey, we migrated the feature requests to discussions. When migrating, GitHub apparently closes the original.

The feature request is still not closed, if that is what your concern was. Our hope is that since in the Discussions the upvotes are shown at the top level by GitHub, we can keep our feature requests here and use the community upvotes to get some sense of which features to pick first. More information is here - https://github.com/ente-io/ente/discussions/480. And this move was done yesterday as part of open sourcing our server.

**Reply by ghost** [reply]
Ok thanks for clarifying

**Comment by seqb** [comment]
That would be a great and helpful feature if we could export a PDF like 0nullpointer wrote. And it would feel more safe as the current text export which can be only imported in Ente Auth, or? If it would help we would sponsor the work therefor.
